docs/_user_guide/best_practices_monitor_usage.md

```diff
@@ -1,116 +1,101 @@
-#  How Do I Check my Resource Usage on the Yens?
-If you would like to check your resource usage on the Yens, there are a few tools available.
+# How Do I Check My Resource Usage on the Yens?
+Monitoring your resource usage is important to be a good citizen of the GSB Research Computing community, but also to be able to effectively estimate your needs on scheduled systems.
 
-- `htop` allows you to get an overview of the activity on the whole system that you are logged into. Furthermore, you can inspect your own processes by typing `htop -u <SUNetID>`. A lot more information about how to decipher the information produced with this command can be found on [this page](/yen/community.html).
-- `topbyuser` is another tool written by us that lists out the active users on the Yen you are logged into and the amount of resources they are currently using.
+These tools can help you characterize your usage:
 
-!!! important
-    We have [community guidelines](/yen/community.html) to illustrate responsible use of our shared resources that our users are expected to follow. These guidelines ensure that the servers can remain accessible to everyone.
+- **`htop`** allows you to get an overview of the activity on the whole system that you are logged into. 
```

Comment:
I would use "node" instead of system since htop only shows per node activity. System can be misinterpreted as all of the yens